webv2/README.md

```diff
@@ -0,0 +1,504 @@
+# HarbourBridge: Schema Assistant (alpha release)
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and
+migration. This README provides details of the HarbourBridge schema assistant,
+which supports schema customization. For general HarbourBridge information see
+this [README](https:githubcomcloudspannerecosystemharbourbridge). Note that the
+schema assistant is currently in alpha release.
+
+### Starting web server for HarbourBridge
+
+The following examples assume `harbourbridge` alias has been setup as
+following.
+
+```sh
+git clone https://github.com/cloudspannerecosystem/harbourbridge
+cd harbourbridge
+alias harbourbridge="go run github.com/cloudspannerecosystem/harbourbridge"
+```
+
+HarbourBridge's Web API feature can be used with all the driver modes available,
+using mysql or postgres dump or direct connection.
+
+To start HarbourBridge web server, run:
+
+```sh
+harbourbridge --web
```

Comment:
From the main.go, shouldnt this be --webv2?

Reply:
updated